components/automate-chef-io/content/docs/compliance.md

```diff
@@ -180,7 +184,7 @@ More Information
 
 ## Profiles
 
-Use the compliance reporting **Profiles_ tab lets you examine the compliance profiles installed under your individual user account.
+Use the compliance reports **Profiles_ tab lets you examine the compliance profiles installed under your individual user account.
```

Comment:
This formatting fix should fix formatting issues throughout the rest of the document.
```suggestion
Use the compliance reports **Profiles** tab lets you examine the compliance profiles installed under your individual user account.
```